Configuration-Interaction/helper_CI.py

```diff
@@ -0,0 +1,366 @@
+"""
+Helper Classes for Configuration Interaction methods
+
+Requirements:
+SciPy 0.13.0+, NumPy 1.7.2+
+
+References:
+Equations from [Szabo:1996]
+"""
+
+__authors__    = "Tianyuan Zhang"
+__credits__   = ["Tianyuan Zhang", "Jeffrey B. Schriber", "Daniel G. A. Smith"]
+
+__copyright__ = "(c) 2014-2017, The Psi4NumPy Developers"
+__license__   = "BSD-3-Clause"
+__date__      = "2017-05-26"
+
+from itertools import combinations
+
+class Determinant:
+    '''
+    class for determinant stored in bits
```

Comment:
Might also want to detail (very briefly) how the bit operations work in python `^,&,etc`.